homeassistant/components/rainmachine/__init__.py

```diff
@@ -386,14 +389,16 @@ def device_class(self):
     def device_info(self):
         """Return device registry information for this entity."""
         return {
+            'config_entry_id': self.rainmachine.config_entry_id,
+            'connections': {
+                (CONNECTION_NETWORK_MAC, self.rainmachine.client.mac)
```

Comment:
This seems to be the same identifier used for the device itself and wouldn't need to be specified as a connection.